src/chapters/booleans-and-conditionals/conditionals.rst

```diff
@@ -0,0 +1,233 @@
+Conditionals
+============
+
+.. index:: ! conditional
+
+A the beginning of this chapter, we motivated our exploration of booleans in JavaScript by observing that having a formal notion of truth and falsehood would allow us to write code that executes conditionally. In other words, we want to be able to write code that only executes when a given condition is ``true``.
+
+Again, here is our motivating example:
+
+.. admonition:: Example
+
+   Consider a banking application that can remind you when a bill is due. The application will notify the user that a bill is due soon, but *only if* the bill has not already been paid.
+
+We summarized the condition as follows: Send a notification of an upcoming bill if the statement "the bill is unpaid" is true.
+
+In such a program, booleans would allow us to write JavaScript that represents the conditional "the bill is unpaid", and based on the truth of such a statement, execute or skip code that notifies the user of an upcoming due date.
```

Comment:
In such a program, JavaScript uses booleans to represent the conditional "the bill is unpaid".  Based on the truth of this statement, the program executes or skips the code for notifying the user.